PCI-DSS profile for fedora

yuumasato opened the pull request:
#### Description:

- Add PCI-DSS profile to Fedora
- The Profile is almost in parity with RHEL7
  - The only difference is check and fix for`smartcard_auth` rule, which is missing in Fedora.

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/OpenSCAP/scap-security-guide/inspections/8e0d609e-d985-4991-8e3f-1705bcf7a999) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/OpenSCAP/scap-security-guide/inspections/8e0d609e-d985-4991-8e3f-1705bcf7a999) completed: **No new issues**

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/OpenSCAP/scap-security-guide/inspections/0ec80f62-6587-49b0-af01-99fca9722cd8) was created.

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/OpenSCAP/scap-security-guide/inspections/9736e6b5-e51b-4ac6-9b20-a055ff663e13) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/OpenSCAP/scap-security-guide/inspections/9736e6b5-e51b-4ac6-9b20-a055ff663e13) completed: **No new issues**

yuumasato commented:
This is close to completion, there still some rules that I need to check the remediation, but the results look good. 
Here are scan reports of Fedora 28:
* before remediation: [pci-dss-fedora.html.txt](https://github.com/OpenSCAP/scap-security-guide/files/2324643/pci-dss-fedora.html.txt)
* after remediation: [remediated-pci-dss-fedora.html.txt](https://github.com/OpenSCAP/scap-security-guide/files/2324644/remediated-pci-dss-fedora.html.txt)

redhatrises commented on linux_os/guide/system/software/updating/ensure_fedora_gpgkey_installed/rule.yml:
```diff
@@ -0,0 +1,50 @@
+references:
+    stigid@rhel6: RHEL-06-000008
+    srg@rhel6: SRG-OS-000090
+    disa@rhel6: '351'
```
Lines 25-26 and 29-31 shouldn't exist.

redhatrises commented on linux_os/guide/system/software/updating/ensure_fedora_gpgkey_installed/rule.yml:
```diff
@@ -0,0 +1,50 @@
+    nist: CM-5(3),SI-7,MA-1(b)
+    ospp@rhel7: FAU_GEN.1.1.c
+    pcidss: Req-6.2
+    srg: "366"
```
This srg doesn't look right to me.

yuumasato replied:
@redhatrises I don't know what to put there, nor where to get the right value, so I'll be removing the `srg` field.

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/OpenSCAP/scap-security-guide/inspections/e580767f-36d2-4ca0-bbb4-fd80544605cd) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/OpenSCAP/scap-security-guide/inspections/e580767f-36d2-4ca0-bbb4-fd80544605cd) completed: **No new issues**

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/OpenSCAP/scap-security-guide/inspections/26d2bdb9-9bb9-4878-a62b-9166095d9853) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/OpenSCAP/scap-security-guide/inspections/26d2bdb9-9bb9-4878-a62b-9166095d9853) completed: **No new issues**

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/OpenSCAP/scap-security-guide/inspections/05a84824-fafd-4d0d-b9c2-0d8826638b12) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/OpenSCAP/scap-security-guide/inspections/05a84824-fafd-4d0d-b9c2-0d8826638b12) completed: **No new issues**

redhatrises commented:
@yuumasato is this no longer a work in progress?

yuumasato commented:
@redhatrises There are some issues with remediation that I would like to iron out, so I guess this is stil WIP, sorry.

In the mean time, check scan results of a Fedora 28 VM after remediation:
[xccdf_org.ssgproject.content_profile_pci-dss-remediation.html.renamed.txt](https://github.com/OpenSCAP/scap-security-guide/files/2331910/xccdf_org.ssgproject.content_profile_pci-dss-remediation.html.renamed.txt)

yuumasato commented:
@redhatrises actually, I think we can work on fixing remediations later.

redhatrises commented:
@yuumasato great job! Ack.
